examples/minimal/packages/client-react/src/mud/setupNetwork.ts

```diff
@@ -44,28 +45,20 @@ export async function setupNetwork() {
     startBlock: BigInt(networkConfig.initialBlockNumber),
   });
 
-  // Request drip from faucet
-  if (networkConfig.faucetServiceUrl) {
-    const address = burnerAccount.address;
-    console.info("[Dev Faucet]: Player address -> ", address);
+  try {
+    console.log("creating faucet client");
+    const faucet = createFaucetClient({ url: "http://localhost:3002/trpc" });
 
-    const faucet = createFaucetService(networkConfig.faucetServiceUrl);
-
-    const requestDrip = async () => {
-      const balance = await publicClient.getBalance({ address });
-      console.info(`[Dev Faucet]: Player balance -> ${balance}`);
-      const lowBalance = balance < parseEther("1");
-      if (lowBalance) {
-        console.info("[Dev Faucet]: Balance is low, dripping funds to player");
-        // Double drip
-        await faucet.dripDev({ address });
-        await faucet.dripDev({ address });
-      }
+    const drip = async () => {
+      console.log("dripping");
+      const tx = await faucet.drip.mutate({ address: burnerAccount.address });
+      console.log("got drip", tx);
```

Comment:
should we still only do this if the balance is below a certain threshold?

Reply:
this is just the example app, but yes the templates should once we get this wired up to k8s

Reply:
or better yet, consider putting the threshold in the faucet itself? so folks don't try to load up